lib/astarte_data_access/config.ex

```diff
@@ -38,4 +57,135 @@ defmodule Astarte.DataAccess.Config do
     os_env: "CASSANDRA_NODES",
     type: CQExNodes,
     default: [{"localhost", 9042}]
+
+  @envdoc "The username used to log into cassandra. Defaults to 'cassandra'."
```

Comment:
Use double quotes around the `"cassandra" default, single quotes are for Erlang charlists